mart/attack/gain.py

```diff
@@ -20,14 +22,14 @@ class RoIHeadTargetClass(Gain):
     """The gain function encourages logits being classified as a particular class, e.g. background
     (class_index==0 in RCNN)."""
 
-    def __init__(self, class_index=0, targeted=True) -> None:
+    def __init__(self, class_index: Optional[int] = 0, targeted: Optional[bool] = True) -> None:
         super().__init__()
 
         self.gain = torch.nn.CrossEntropyLoss()
         self.class_index = class_index
         self.targeted = targeted
 
-    def forward(self, roi_heads_class_logits, proposals):
+    def forward(self, roi_heads_class_logits: torch.Tensor, proposals: Any) -> torch.Tensor:
```

Comment:
I think `proposals` should be a `Tensor`?

https://github.com/pytorch/vision/blob/0dceac025615a1c2df6ec1675d8f9d7757432a49/torchvision/models/detection/rpn.py#LL243C27-L243C27